### PR TITLE
fix notify length issue

### LINKED ISSUE #96: Notify command joins user tags with extra comma
**Describe the bug**
Each message with user tags will always end with a comma. It should look a bit more polished if that's not the case.

**To Reproduce**
1. Run a notify command with people subbed to that list

**Expected behavior**
No trailing comma

### PR DESCRIPTION
**Pull request by bobvdvleuten**
quick fix to split the notify list of users into multiple messages when necessary

### CODE REVIEW

**Review by Jan-PieterBaert — COMMENTED** [review]
Looks good

**Review by Autom3 — CHANGES_REQUESTED** [review]
Each message will always end with a comma using this implementation. It should look a bit more polished if that's not the case

**Comment by bobvdvleuten** [comment]
> 
> 
> Each message will always end with a comma using this implementation. It should look a bit more polished if that's not the case

indeed but it's a quick patch for now. Long term should be fixe dindeed

**Review by Autom3 — DISMISSED** [review]

File: Helpers/constants.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 NOTIFY_EMBED_COLOR = 0x0a5bf3
 NOTIFY_MAX_MSG_LENGTH = 1024 #characters
+DISCORD_MAX_MSG_LENGTH = 2048
```
Comment (Eva774) [inline Helpers/constants.py]: The maximum message length is 2000 characters and not 2048.

**Review by Eva774 — CHANGES_REQUESTED** [review]

**Review by Eva774 — APPROVED** [review]

**Review by Autom3 — APPROVED** [review]

File: Commands/notify.py [hunk 2]
```diff
@@ -138,7 +142,8 @@ async def notify(self, ctx: commands.Context, list_name: str, *, message: typing
             )
 
         await ctx.channel.send(embed=embed)
-        await ctx.send(users_str)
+        for users_str in user_messages:
+            await ctx.send(users_str)
```
Comment (Jan-PieterBaert) [inline Commands/notify.py:146]: To solve #96 you could run rstrip on the `users_str` to cut of the `, ` or you could slice those 2 chars away

**Review by woutermeuwis — APPROVED** [review]

**Pull request merged**